app/formSchema/uiSchema.ts

```diff
@@ -59,14 +59,68 @@ const uiSchema = {
     'hasPassiveInfrastructure',
     'isInfrastuctureAvailable',
     'requiresThirdPartyInfrastructureAccess',
+    'geographicArea',
     'projectSpanMultipleLocations',
+    'provincesTerritories',
     'projectLocations',
     'geographicAreaDescription',
     'projectDescription',
     'totalEligbleCosts',
     'totalProjectCost',
     'requestedCCBCFunding',
   ],
+  projectTitle: {
```

Comment:
in the uiSchema you'll need to specify that isAuthContactSigningOfficer is of ui:widget: RadioWidget

Reply:
as well as isAltContactSigningOfficer for ui:widget: RadioWidget